specification/grammars/annotation.lark

```diff
@@ -36,7 +36,15 @@ ORDINAL : (DIGIT)+
 
 NUMBER : DIGIT ("." (DIGIT)+)?
 
-AMINO_ACID : "A" | "R" | "N" | "D" | "C" | "E" | "Q" | "G" | "H" | "K" | "M" | "F" | "P" | "S" | "T" | "W" | "Y" | "V" | "I" | "L"
+AMINO_ACID : UPPER_CASE_LETTER
+
+ELEMENT : "Ac" | "Ag" | "Al" | "Am" | "Ar" | "As" | "At" | "Au" | "B" | "Ba" | "Be" | "Bh" | "Bi" | "Bk" | "Br" | "C" | "Ca" | "Cd" |
+          "Ce" | "Cf" | "Cl" | "Cm" | "Cn" | "Co" | "Cr" | "Cs" | "Cu" | "Db" | "Ds" | "Dy" | "Er" | "Es" | "Eu" | "F" | "Fe" | "Fm" |
+          "Fr" | "Ga" | "Gd" | "Ge" | "H" | "H+" | "He" | "Hf" | "Hg" | "Ho" | "Hs" | "I" | "In" | "Ir" | "K" | "Kr" | "La" | "Li" |
+          "Lr" | "Lu" | "Md" | "Mg" | "Mn" | "Mo" | "Mt" | "N" | "Na" | "Nb" | "Nd" | "Ne" | "Ni" | "No" | "Np" | "O" | "Os" | "P" |
+          "Pa" | "Pb" | "Pd" | "Pm" | "Po" | "Pr" | "Pt" | "Pu" | "Ra" | "Rb" | "Re" | "Rf" | "Rg" | "Rh" | "Rn" | "Ru" | "S" | "Sb" |
+          "Sc" | "Se" | "Sg" | "Si" | "Sm" | "Sn" | "Sr" | "Ta" | "Tb" | "Tc" | "Te" | "Th" | "Ti" | "Tl" | "Tm" | "U" | "Uuh" | "Uuo" |
```

Comment:
Un-un-octium has gotten its name of Ogassium, although this is very minor for any proteomics use cases.